command/ssh.go

```diff
@@ -45,6 +47,15 @@ func (c *SSHCommand) Run(args []string) int {
 	if err := flags.Parse(args); err != nil {
 		return 1
 	}
+
+	if os.Getenv("VAULT_STRICT_HOST_KEY_CHECKING") != "" {
```

Comment:
You are right. Done!

 I had referred to `DEV_ROOT_TOKEN_ID` way of doing things. It looks like that can take a change as well. If you agree I will give precedence to flags for `devRootTokenID` and `devListenAddress`, in a follow up PR.